CONTRIBUTING.md

```diff
@@ -79,6 +81,16 @@ $ pre-commit install
 
 :tada: Now you're ready to create a new branch, add a feature or fix a bug, then send us a pull request! :tada:
 
+### Note about Apple ARM-based chips
+
+To run the SDK on an Apple ARM chip, upgrade your Python version to 3.8 or higher and use the latest version of pip.
```

Comment:
Do we need to add sth like this to remind user to use python 3.8 or higher in the `readme.md`?

Reply:
and use `0.41.0` or higher version of the sdk